src/website/index.html

```diff
@@ -176,20 +176,39 @@
         }, pubKey, str2ab(atob(signature)), new TextEncoder("utf-8").encode(data))
   }
 
-  function onScanSuccess(decodedText, decodedResult) {
+  async function onScanSuccess(decodedText, decodedResult) {
     const data = JSON.parse(decodedText)
 
     const signature = data.signature
 
     delete data.signature
 
-    verifySignature(signature, JSON.stringify(data)).then(res => {
+    try {
+      const res = await verifySignature(signature, JSON.stringify(data))
       if (!res) {
         alert('Signature is invalid.')
         return
       }
+
+      const response = await fetch('https://a50jc5jboe.execute-api.us-east-1.amazonaws.com/kuberoke-kc-2023-chi-prod/attendance', {
+        method: "POST",
+        mode: "cors",
+        cache: "no-cache",
+        headers: {
+          "Content-Type": "application/json",
+          "X-API-Key": API_KEY
+        },
+        redirect: "follow",
+        referrerPolicy: "no-referrer",
+        body: JSON.stringify({
+          email: data.email
+        })
+      })
```

Comment:
Agreed. I have moved this out of that flow into its own. Now we render the QR code twice, which is not great, but to avoid it would require chaining the functions(?) so onScanSuccess would need to return the data I think.